### ISSUE TITLE
Overall: issues with how concepts are introduced for PyTorch

### ISSUE THREAD

**Issue by ashwinvis** [mimer-ai/deep-learning-intro#15]
- Hard to talk model training without talking about gradient descent, or even batches in Ep. 2
